EnglishTranslation-master/ETC.tsv

```diff
@@ -19574,21 +19574,21 @@ ETC_20151224_019574	You've completed {DAYPLAYCOUNT} times out of {DAYMAXCOUNT} t
 ETC_20151224_019575	Charging the launcher
 ETC_20151224_019576	Purified the evil spirit!
 ETC_20151224_019577	You receive a beneficial effect because you purified the evil spirit!
-ETC_20151224_019578	You have collected the enough Mushroom
-ETC_20151224_019579	The monsters in Gogal District 2 has weakened!
-ETC_20151224_019580	The monsters in Gogal District 1 has weakened!
+ETC_20151224_019578	You have collected enough Mushrooms
+ETC_20151224_019579	The monsters in Exhaustion District 2 have weakened!
+ETC_20151224_019580	The monsters in Exhaustion District 1 have weakened!
 ETC_20151224_019581	The debuff effect wore off
 ETC_20151224_019582	The door to the left secret room has opened
 ETC_20151224_019583	The door to the right secret room has opened
-ETC_20151224_019584	Controlling the Device
+ETC_20151224_019584	The device has been activated
```

Comment:
So this line is from Nevellet Quarry, the 2f puzzle quest. Once you kill enough mobs near one of the two doors, the door opens, and there's torch behind the door. This line is a map wide broadcast when you light one of the torches. In this context, 'controlling' doesn't make much sense.